docs/security/upgrading/upgrade-to-ubuntu-16-04.md

```diff
@@ -98,7 +98,7 @@ Remember to perform these steps in a Lish or Glish session:
 
 3.  You're now ready to begin the upgrade to Ubuntu 16.04 LTS:
 
-        do-release-upgrade -d
+        do-release-upgrade
```

Comment:
@navjotjsingh ,
Ran through it again to confirm and check on that "No new release found" error (screenshot attached).
After some further research, I found that you're right, but not yet.
The -d flag calls for a development release because Ubuntu 14.04 doesn't recognize 16.04 as an upgrade path until 16.04.1 is released (at some point in July). I'm going to leave the guide as written until 16.04.1 is released, then we'll update to remove the flag.

![1__ssh_and_how_to_upgrade_to_ubuntu_16_04_lts](https://cloud.githubusercontent.com/assets/17991901/15944765/151f1700-2e5d-11e6-9d2a-3d98f3fe7df7.png)

Thanks,
Edward